pkg/model/bootstrapscript.go

```diff
@@ -71,6 +71,17 @@ func (b *BootstrapScript) ResourceNodeUp(ig *kops.InstanceGroup) (*fi.ResourceHo
 			}
 			return ""
 		},
+
+		// Pass in configuration for the cluster proxy
+		"ProxyEnv": func() string {
+			if os.Getenv("CLUSTER_HTTP_PROXY") != "" || os.Getenv("CLUSTER_HTTPS_PROXY") != "" {
+				return fmt.Sprintf("export HTTP_PROXY=%s\nexport HTTPS_PROXY=%s\nexport NO_PROXY=%s\n",
+					os.Getenv("CLUSTER_HTTP_PROXY"),
```

Comment:
So my understanding is that we would have to set these env vars every time we called `kops update`.

I think this could instead be a couple of fields; it could either be a new top-level object (like `authorization`) or we could put it into the `networking` object.  I think my preference would be a new top level object, because the `networking` object is currently an enum-style, where we only expect a single value to be set.

If you agree I'm happy to assist / write a doc & WIP PR on how to do this - let me know.